LICENSE-binary

```diff
@@ -251,6 +252,7 @@ org.apache.commons:commons-crypto
 org.apache.commons:commons-lang3
 org.apache.hadoop:hadoop-client-api
 org.apache.hadoop:hadoop-client-runtime
+org.apache.ibatis:mybatis
```

Comment:
this is a NOTICE file inside mybatis jar, the NOTICE-binary should be updated correspondingly,